application/config.json.template

```diff
@@ -90,4 +90,9 @@
     "logErrorChannelWebhook": "<put_your_webhook_here>",
     "openaiApiKey": "<check pins in #tjbot_discussion for the key>",
     "sourceCodeBaseUrl": "<https://github.com/<your_account_here>/<your_repo_here>/blob/master/application/src/main/java/>"
+    "jshell": {
+        "baseUrl": "http://localhost:8080/jshell/",
```

Comment:
its better to leave it out entirely if we cant have a meaningful default.
so `<put_your_api_endpoint_here>`

Reply:
Since the api should be on the same machine, and should have port 8080 by default, it totally makes sense to have this default

Reply:
the config template is not for the production machine. it is for the developer. and the developer does not run a second program when developing tj-bot. in fact, there is a high risk that some other thing is currently running on their machine on 8080.

if you can not provide an external link that is available out-the-box when a dev runs tj-bot locally, u shouldnt put sth meaningful there and the feature should be disabled. (similar to for example the discord log-forwarder)